self_supervised_pretraining/ssl_finetune.ipynb

```diff
@@ -5,23 +5,50 @@
    "id": "14fc43c2",
    "metadata": {},
    "source": [
+    "# Finetune 3D Multi-Class Abdominal Segmentation Model Using SSL Pre-trained Weights\n",
     "Copyright (c) MONAI Consortium  \n",
     "Licensed under the Apache License, Version 2.0 (the \"License\");  \n",
     "you may not use this file except in compliance with the License.  \n",
     "You may obtain a copy of the License at  \n",
-    "    http://www.apache.org/licenses/LICENSE-2.0  \n",
+    "&nbsp;&nbsp;&nbsp;&nbsp;http://www.apache.org/licenses/LICENSE-2.0  \n",
     "Unless required by applicable law or agreed to in writing, software  \n",
     "distributed under the License is distributed on an \"AS IS\" BASIS,  \n",
     "WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  \n",
     "See the License for the specific language governing permissions and  \n",
-    "limitations under the License.  \n",
-    "\n",
+    "limitations under the License."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "26205edd",
+   "metadata": {},
+   "source": [
     "##### Note:\n",
     "[Beyond the cranial vault](https://www.synapse.org/#!Synapse:syn3193805/wiki/217789) abdominal segmentation multi-class dataset needs to be downloaded for the execution of this notebook. Downloading data to the `data_root` path defined variable is optimal, if data is downloaded at another location please change the path `data_root` accordingly. Ensure to use the prior logdir path from ssl_train notebook for defining the `pretraining_path` in this notebook if pretrained weights are to be used.\n",
     "\n",
     "Please see the Readme for more details about the dataset. Multiple data splits have already been provided in subsets of 6, 12 and 24 (full data) with consistent validation splits.\n",
     "\n",
-    "##### Import statements"
+    "## Setup Environment"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "42b4e3cb",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!python -c \"import monai\" || pip install -q \"monai-weekly[pillow, tqdm]\"\n",
+    "!python -c \"import matplotlib\" || pip install -q matplotlib\n",
+    "%matplotlib inline"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "88f7f858",
+   "metadata": {},
+   "source": [
+    "## Import statements"
```

Comment:
Can you change this to “## Setup imports" as other notebooks? Thanks!

Reply:
Also it misses the following required check:
```
from monai.config import print_config
....
print_config()`
```

Reply:
Addressed